src/api/pastel-rpc/rpc.ts

```diff
@@ -8,7 +9,7 @@ export type TRPCConfig = {
 
 export async function rpc<T>(
   method: string,
-  params: string[],
+  params: any[],
```

Comment:
Why would this be changed? RPC calls accept string arguments. This doesn't seem right